Update/kage_009.txt

```diff
@@ -162,32 +162,32 @@ void main()
 //rそれに、自分が責められるのならまだいい。…大石が垣内署の山沖署長の好意で捜査活動を行っている以上、下手な突出は彼らにも迷惑をかけることにもなりかねない。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "それに、自分が責められるのならまだいい。…大石が垣内署の山沖署長の好意で捜査活動を行っている以上、下手な突出は彼らにも迷惑をかけることにもなりかねない。",
-			NULL, "Plus, it allowed them to pin the blame on others. ...Ooishi-san was doing investigative work with Kakiuchi Station's Chief Yamaoki, but anything coming out of that could cause problems.", Line_Normal);
+			NULL, "Plus, it would still be fine if he could blame himself. ...But as long as he was doing this investigation work by grace of the good will of Kakiuchi Station's Chief Yamaoki, anything wrong moves Ooishi made would likely cause problems for the station.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Plus, it would still be fine if he could blame himself. ...But as long as he was doing this investigation work through the good will of Kakiuchi Station's Chief Yamaoki, any wrong move Ooishi made would likely cause problems for the station.", Line_Normal);
```